src/go/configgenerator/route_generator.go

```diff
@@ -49,12 +50,20 @@ func MakeRouteConfig(serviceInfo *configinfo.ServiceInfo) (*routepb.RouteConfigu
 		Domains: []string{"*"},
 	}
 
-	// Per-selector routes for both local and remote backends.
-	brRoutes, err := makeRouteTable(serviceInfo)
+	// The router matches request by sequence of routes and the first matched one will be picked, so the order of routes matters.
```

Comment:
Router always selects the first matched routes, so the order is important.

Reply:
Done.